tutorials/desktop/firefox-for-web-developers/firefox-for-web-developers.md

```diff
@@ -0,0 +1,309 @@
+---
+id: firefox-for-web-developers
+summary: Learn how to access the Firefox web developer tools, and how to install Developer Edition of the Firefox browser
+categories: desktop
+tags: tutorial, desktop, web, developer, Firefox, HTML, CSS, JavaScript
+difficulty: 2
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2018-06-27
+author: Canonical Web Team <webteam@canonical.com>
+
+---
+
+# Firefox for web developers
+
+## Overview
+Duration: 1:00
+
+In this tutorial you will learn how to access the web development tools that are built into every copy of Firefox. You will also learn how to install the Firefox Developer Edition alongside your normal Firefox installation, allowing you to keep development work separate from your normal day-to-day browsing.
+
+
+### What you'll learn
+* How to access the developer tools
+* How to use the inspector and console tools
+* How to configure the user interface to show the tools you want
+* Why you might want to install a second browser for development
+* How to install Firefox Developer Edition
+
+
+### What you'll need
+* A computer running Ubuntu 16.04 or above
+* The Firefox web browser (installed by default in Ubuntu desktop systems)
+
+
+negative
+: **Breaking things is easy… but so is fixing them**
+Using the developer tools can render a web page unusable. Don't worry if you do break something, just refresh the browser by pressing **F5** to reload the page in its original state.
```

Comment:
I really like that you've mentioned this. Is there a way to make it clear that you're not breaking the *actual* website - only your view of it? Maybe something like

> Although nothing you do in developer tools can possibly affect the website itself, using the developer tools can render a web page unusable in your browser.

What do you think?

Reply:
Good point. I'll tweak that paragraph to make it clearer.